lib/wfsc/falco_est_pairwise_probing.m

```diff
@@ -232,7 +231,8 @@
         %--Generate the DM command map for the probe
         switch lower(mp.estimator)
             case{'pairwise-rect', 'pwp-bp', 'pwp-kf'} 
-                probeCmd = falco_gen_pairwise_probe(mp, InormProbe, probePhaseVec(iProbe), iStar, mp.est.probe.rotation);

```

Comment:
@davidsmarx Can you update the branch to use the latest version of `falco_gen_pairwise_probe()` that requires the rotation? That's the default now to need it. I'm really confused why Git didn't do that automatically when you merged in master somewhat recently.